src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -74,6 +74,9 @@ class ReimbursementAccountPage extends React.Component {
 
         // If we are trying to navigate to `/bank-account/new` and we already have a bank account then don't allow returning to `/new`
         BankAccounts.fetchFreePlanVerifiedBankAccount(stepToOpen !== CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT ? stepToOpen : '');
+
+        // If we are coming back from the Plaid OAuth flow
+        this.receivedRedirectURI = BankAccounts.getOAuthReceivedRedirectURI();
```

Comment:
Curious why this is in `componentDidMount()` instead of the `render()` and used directly?

Reply:
My thoughts: we only need to grab this `redirectURI` once when we return from the Plaid OAuth flow to the `bank-accounts` route. Adding it to the `render()` method means we'll be calling it each time we move from one setup step to the next, which is unnecessary.

Reply:
>Adding it to the render() method means we'll be calling it each time we move from one setup step to the next, which is unnecessary

Unnecessary, but maybe convenient and harmless?

Is the intention that once we set this variable it will be used when the component renders a second time? 

My understanding is that `componentDidMount()` occurs _after_ the first call to `render()` so my expectation is that `this. receivedRedirectURI` would be `undefined` the first time this component renders.

Reply:
> Unnecessary, but maybe convenient and harmless?

Yeah, agree here. 

> Is the intention that once we set this variable it will be used when the component renders a second time?

That was my initial thought, but on consideration I think it is better to call it within the `render()` method.